pkg/ccl/changefeedccl/sink_webhook.go

```diff
@@ -300,7 +300,7 @@ func makeWebhookSink(
 		exitWorkers: cancel,
 		parallelism: parallelism,
 		ts:          source,
-		metrics:     m,
+		metrics:     mb(true),
```

Comment:
could be localhost, but I guess that's fine.
I commented elsewhere, but I think it would be nice to add constants to require/opt-out of resource acccounting.